pkg/crawl/report.go

```diff
@@ -33,6 +33,10 @@ type Reporter struct {
 	DisableCVE45105 bool
 	// Number of issues that have been found
 	count int64
+	// The tags of the image currently being reported
+	imageTags []string
```

Comment:
This is probably okay for now, but don't think this is great design/construction -- the Docker layer is meant to be a wrapper around the filesystem layer, but now that abstraction is seeping into the reporter. A proper design would probably require another level of wrapping or more generic metadata so this is probably fine on balance, but flagging as a code smell.